src/codeowners/generate.ts

```diff
@@ -50,6 +50,23 @@ function generateLine(
       (pCSTLine.inlineComment ? ` #${pCSTLine.inlineComment}` : "")
     );
   }
+  if (pCSTLine.type === "section") {
+    const lUserNames = uniq(
+      pCSTLine.users.flatMap((pUser) => expandTeamToUserNames(pUser, pTeamMap)),
+    )
```

Comment:
this line is a duplicate. not per  se an issue if it's the clearest way to  express this. Is it indeed?

Reply:
looks like it's better slightly de-duplicated idd.